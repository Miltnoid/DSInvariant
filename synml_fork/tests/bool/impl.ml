type bool =
  | True
  | False

let impl : bool -> bool -> bool |>
  { True => True => True
  ; True => False => False
  ; False => True => True
  ; False => False => True } = ?
