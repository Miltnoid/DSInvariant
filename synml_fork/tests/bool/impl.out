(* Generated Mon Dec 29 00:38:04 2014 *)
let impl : bool -> bool -> bool =
  fun (b1:bool) -> fun (b2:bool) -> match b1 with
                                      | True -> b2
                                      | False -> True
;;
